GoogleDataTransport/GDTCORLibrary/GDTCORFlatFileStorage.m

```diff
@@ -539,6 +539,40 @@ - (void)storageSizeWithCallback:(void (^)(uint64_t storageSize))onComplete {
   return [eventPathsForBatchID copy];
 }
 
+/** Makes a copy of the contents of a directory to a directory at the specified path.*/
+- (BOOL)moveContentsOfDirectoryAtPath:(NSString *)sourcePath
```

Comment:
Why doesn't `moveItem` meet the requirements - https://developer.apple.com/documentation/foundation/filemanager/1413529-moveitem

Reply:
Added clarification comments to the code. The following `moveItem` behaviour makes it failing for the case:
```
...If an item with the same name already exists at dstPath, this method stops the move attempt and returns an appropriate error...
```